scripts/jenkins-yoctobuild-build.sh

```diff
@@ -462,6 +462,7 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
             mender-conductor-enterprise)
                 cd go/src/github.com/mendersoftware/$build
                 docker build --build-arg REVISION=pr -t mendersoftware/mender-conductor-enterprise:pr ./server
+                docker build -t mendersoftware/org-welcome-email-preparer:pr ./workers/prepare_org_welcome_email
                 $WORKSPACE/integration/extra/release_tool.py --set-version-of $build --version pr
```

Comment:
Why don't we need to set with release_tool the version of org-welcome-email-preparer to PR?

Reply:
I suppose we could, for completeness sake, but it's not necessary because the repository and the docker image have the same name. Therefore the release tool sets the version of all containers which belong to the *git* repository "mender-conductor-enterprise", and this includes "org-welcome-email-preparer".

As long as this lives under the `mender-conductor-enterprise)` case selector, I think I'll leave it as is.

Reply:
:+1: